### DISCUSSION TITLE
Works great for controlling Waveshare Modbus IP relay device

### DISCUSSION THREAD

**Discussion by krumble1** [Zayik/CommandSender#17] [General]
This is working great to control the [Waveshare Modbus POE ETH Relay](https://www.waveshare.com/wiki/Modbus_POE_ETH_Relay) I ordered on Amazon. Thank you so much for making this!

If anyone has questions about getting that working with a Stream Deck, feel free to message me here and hopefully I'll see it!

**Comment by Zayik** [comment]
Thanks. Glad it's working great for you!
